<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="titleWid" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>智能负载柜 </string>
     </property>
    </widget>
    <widget class="QLabel" name="logLab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/logo.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="vLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>V_0.0.1</string>
     </property>
    </widget>
    <widget class="QToolButton" name="homeBtn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>主界面</string>
     </property>
    </widget>
    <widget class="QToolButton" name="inputBtn">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>10</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>输入位</string>
     </property>
    </widget>
    <widget class="QToolButton" name="setBtn">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>10</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>参数设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="timeBtn">
    <property name="geometry">
     <rect>
      <x>778</x>
      <y>570</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>----</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>661</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWid">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1021</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
